sonar-project.properties

```diff
@@ -1,3 +1,7 @@
 sonar.projectKey=DataBiosphere_terra-ui
 sonar.organization=broad-databiosphere
-sonar.javascript.lcov.reportPaths=coverage/lcov.info
+
+# Only scan code under src.
+sonar.sources=src
```

Comment:
The default value for this is `.` so using `src` should eliminate many files that we don't need to include.